packages/forms/src/widgets/ColumnsWidget/ColumnsWidget.js

```diff
@@ -44,32 +44,42 @@ if (process.env.NODE_ENV !== 'production') {
 	};
 }
 
-function onColumnChange(key, onChange, formData) {
-	return function handleChange(change) {
-		onChange(Object.assign({}, formData, { [key]: change }));
-	};
-}
+export default class ColumnsWidget extends React.Component {
+	constructor(props) {
+		super(props);
+		this.onColumnChange = this.onColumnChange.bind(this);
```

Comment:
you can do that class ColumnsWidget extends React.Component { onColumnChange = () => { ... your body function } }

Reply:
i do a study on that :)

Reply:
for the sake of code base homogeneity we won't.

Reply:
And the sake of debug stacktraces